tests/unit/config/config_test.py

```diff
@@ -3843,8 +3878,71 @@ def test_no_base(self):
 
 class MergeNetworksTest(unittest.TestCase, MergeListsTest):
```

Comment:
A test to check that lists (aliases and link_local_ips) get combined rather than overridden would be good to have as well.

Reply:
Adding